saleor/checkout/forms.py

```diff
@@ -68,7 +68,7 @@ class ShippingCountryChoiceField(forms.ModelChoiceField):
 
     def label_from_instance(self, obj):
         """Return a friendly label for the shipping method."""
-        price_html = format_price(obj.price.gross, obj.price.currency)
+        price_html = format_price(obj.price.amount, obj.price.currency)
```

Comment:
Why does it need two parameters? It looks like this should only accept a `Money` instance?